### DISCUSSION TITLE
Making letter keys auto-scroll to those names in Change Hero menu

### DISCUSSION THREAD

**Discussion by HauntedEri** [Heroes-Profile/heroesprofile#925] [Ideas]
On the old version of the site, when you were changing selection in the "Change Hero" menu dropdown, you could press a letter key on the keyboard and it would instantly scroll to the first hero (alphabetically) with that letter starting their name. I would love to have this functionality back, as I used it quite a bit!

**Comment by Zemill** [comment]
I didn't even realize the old site did this.   We did add a search for the heroes dropdowns but I imagine it doesn't key off hitting a keyboard key until you click inside it.  


I will look into this

**Comment by Zemill** [comment]
This will be added back in when https://github.com/Heroes-Profile/heroesprofile/pull/1071 is merged
